hm10_3.py

```diff
@@ -0,0 +1,61 @@
+""" Task 3: function arguments are positive numbers """
+
+
+def typed(*arg_types):
+
+    """ Function takes arguments _any """
+
+    def decorator(func):
+
+        """ Function takes arguments func """
+
+        def wrapper(*args):
+
+            """ Function takes arguments in any value """
+
+            converted_args = []
+            for arg, arg_type in zip(args, arg_types):
+                converted_arg = arg_type(arg)
+                converted_args.append(converted_arg)
+            return func(*converted_args)
+        return wrapper
+    return decorator
+
+
+@typed(str, int, str)
```

Comment:
the requirements say that only 1 type must be passed
```suggestion
@typed(str) / @typed(int) / ....
```